app/src/renderer/components/governance/PageProposal.vue

```diff
@@ -0,0 +1,196 @@
+            tm-btn(v-if="button === 'deposit'" value="Deposit" color="primary" @click.native="deposit")
+            tm-btn(v-if="!button" disabled value="Deposit / Vote" color="primary")
+
+        .row.description
+          p This {{ proposalType }} proposal ({{ `#` + proposal.proposal_id }}) was submitted at block {{ submitBlock }} and voting started at {{ voteBlock }}.
+
+        .row.validator-profile__header__data.votes
+          dl.colored_dl
+            dt Deposit
+            dd {{ proposal.total_deposit[0].amount + ` ` + proposal.total_deposit[0].denom }}
+          .validator-profile__header__data__break
+          dl.colored_dl
+            dt Yes
+            dd {{ proposal.tally_result.yes }} / {{ yesPercentage }}
+          dl.colored_dl
+            dt No
+            dd {{ proposal.tally_result.no }} / {{ noPercentage }}
+          dl.colored_dl
+            dt No with Veto
+            dd {{ proposal.tally_result.no_with_veto }} / {{ noWithVetoPercentage }}
+          dl.colored_dl
+            dt Abstain
+            dd {{ proposal.tally_result.abstain }} / {{ abstainPercentage }}
+
+    .validator-profile__details.validator-profile__section
+      .column
+        .row
+          text-block(:content="proposal.description")
+</template>
+
+<script>
+import num from "scripts/num"
+import { TmBtn, TmFormStruct, TmPage, TmPart, TmToolBar } from "@tendermint/ui"
+import TmBalance from "common/TmBalance"
+import FieldVote from "common/TmFieldVote"
+import TextBlock from "common/TextBlock"
+export default {
+  name: `page-proposal`,
+  props: [`proposal`],
+  components: {
+    TmBalance,
+    TmBtn,
+    FieldVote,
+    TmFormStruct,
+    TmToolBar,
+    TmPage,
+    TmPart,
+    TextBlock
+  },
+  computed: {
+    proposalType() {
+      return this.proposal.proposal_type.toLowerCase()
+    },
+    submitBlock() {
+      return `#` + num.prettyInt(this.proposal.submit_block)
+    },
+    voteBlock() {
+      if (this.proposal.submit_block === this.proposal.voting_start_block) {
+        return `the same block`
+      } else {
+        return `block #` + num.prettyInt(this.proposal.voting_start_block)
+      }
+    },
+    button() {
+      if (this.proposal.proposal_status === `Passed`) return null
+
+      if (this.proposal.proposal_status === `Rejected`) return null
+
+      if (this.proposal.proposal_status === `Active`) return `vote`
+
+      if (this.proposal.proposal_status === `Pending`) return `deposit`
+    },
+    status() {
+      if (this.proposal.proposal_status === `Passed`)
+        return `This proposal has passed`
+
+      if (this.proposal.proposal_status === `Rejected`)
+        return `This proposal has been rejected and voting is closed`
+
+      if (this.proposal.proposal_status === `Active`)
+        return `Voting for this proposal is open`
+
+      if (this.proposal.proposal_status === `Pending`)
+        return `Deposits are open for this proposal`
```

Comment:
Deposits are always open as long as the proposal is open, in this case the proposal is pending because it hasn't met the minimum deposit

Reply:
why would a user deposit if a proposal has received enough to open voting?

Reply:
I also said that... But you can ask @sunnya97

Reply:
we'll see what sunny says, but i don't think we should block on this.